.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -1,3 +1,12 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: ''
```

Comment:
Maybe we should add bug as default label and add another (almost empty) template for "Questions" and/or "Others"?

Reply:
I like the idea of a template for "'Questions' and/or 'Others'". Will automatically adding "bug" as default confuse us as to which issues have been verified as and actual bug as opposed to something like user error?

Reply:
I thought that "Questions and/or Comments" sounded better, but I'd be open to changing it. I also have the template set to auto label with 'question'.

Reply:
I would say it's OK if bug is set by default, we can work the opposite way (close issue and/or remove bug tag if it wasn't a bug).

Reply:
bug label added